life_expectancy/tests/create_fixture.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+from pathlib import Path
+import numpy as np
+from loaders import load_data, save_data
+from _cleaning import clean_data
+
+
+# BASE_DIR = Path(__file__).parent.parent / "data"
+# FIXTURES_DIR = Path(__file__).parent / "fixtures"
+# print(BASE_DIR)
+# print(FIXTURES_DIR)
```

Comment:
![image](https://github.com/danizao/pod5_/assets/20598571/3297373b-4958-4b7d-a442-cce6841d1f3b)

Reply:
Done! Thanks.

Reply:
Did you push the change?

Reply:
I did now Fernando